packages/orbit-components/src/Accordion/AccordionSection/components/SectionHeader.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+import * as React from "react";
+import styled from "styled-components";
+
+import Button from "../../../Button";
+import defaultTheme from "../../../defaultTheme";
+
+const Wrapper = styled.div`
+  display: flex;
+  padding: ${({ theme, noPadding }) =>
+    !noPadding && `${theme.orbit.spaceXLarge} ${theme.orbit.spaceLarge}`};
+  background-color: white;
```

Comment:
tokens should be used for colors: `background-color: ${({theme}) => theme.orbit.paletteWhite}`